vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor.cc

```diff
@@ -0,0 +1,106 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_info.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events.h"
+
+#include <iostream>
+#include <algorithm>
+
+#include "base/check.h"
+#include "bat/ads/internal/base/logging_util.h"
+#include "bat/ads/internal/ml/data/vector_data.h"
+#include "bat/ads/internal/features/text_embedding_features.h"
+#include "bat/ads/internal/deprecated/client/client_state_manager.h"
```

Comment:
not needed anymore

Reply:
Remove deprecated client_state_manager.h include statement